recipes-server/deploy-server-files/files/php-initscript

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+install -d -m 755 /run/php
+touch /run/php/php-fpm.sock
+chown www-data:www-data /run/php
+chmod 660 /run/php/php-fpm.sock
+systemctl start php-fpm
+systemctl restart nginx
+chown www-data:www-data /run/php/php-fpm.sock
```

Comment:
I was thinking about that last night and I came to a conclusion that it might be caused by php-fpm service, as it's started by root. Maybe we should add sth in the php-fpm.service? e.g all those? in `ExecStartPre`? Then this service won't be necessary.
Please consider.

Reply:
Even if I would be able to enable it in config, this service still will be needed because I need to create this socket, and restart services after the creation. As I saw in your php-fpm service, it requires couple packages that are missing in the target image, so it is the best option for now.